ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/versions/merge/helpers/MiscHelpersMerge.java

```diff
@@ -40,4 +41,8 @@ public boolean isMergeBlock(final BeaconState genericState, final BeaconBlock bl
   public boolean isExecutionEnabled(final BeaconState genericState, final BeaconBlock block) {
     return isMergeBlock(genericState, block) || isMergeComplete(genericState);
   }
+
+  public UInt64 computeTimestampAtSlot(final BeaconState state, final UInt64 slot) {
```

Comment:
nit: `state` -> `genericState`

Reply:
I left this as state matching the spec because we didn't need to have a `BeaconStateMerge` version and thus didn't need to differentiate the names.